recipes/qhull/all/conandata.yml

```diff
@@ -1,27 +1,27 @@
 sources:
+  "cci.20231130":
```

Comment:
We're changing the `cci.YYYYMMDD` scheme to ensure it works in version ranges, I'll submit the suggestion once we get final consensus